acme/acme/client.py

```diff
@@ -152,6 +152,28 @@ def deactivate_authorization(self,
         return self._authzr_from_response(response,
             authzr.body.identifier, authzr.uri)
 
+    # def update_account_key(self,
+    #                        regr: messages.RegistrationResource,
+    #                        response: requests.Response, uri: Optional[str] = None
+    #                        ) -> messages.KeyChange:
+    #     """
```

Comment:
It's WIP. Struggling a bit with it. Would be nice to complete it, either as part of this PR, or a dedicated one.